octavia-cli/octavia_cli/templates/source_or_destination.yaml.j2

```diff
@@ -0,0 +1,79 @@
+# Configuration for {{ definition.docker_repository }}
+# Documentation about this connector can be found at {{ definition.documentation_url }}
+resource_name: {{ resource_name}}
+definition_type: {{ definition.type}}
+definition_id: {{ definition.id }}
+definition_image: {{ definition.docker_repository }}
+definition_version: {{ definition.docker_image_tag }}
+
+{%- macro render_field(field, is_commented) %}
+{%- if is_commented %}# {% endif %}{{ field.name }}:{% if field.default %} {{ field.default | tojson() }}{% endif %} # {{ field.comment }}
+{%- endmacro %}
+
+{%- macro render_sub_fields(sub_fields, is_commented) %}
+{%- for f in sub_fields %}
+{%- if f.type == "object" and not f.oneOf %}
+{{- render_object_field(f)|indent(2, False) }}
+{%- elif f.oneOf %}}
```

Comment:
JsonSchema also supports `anyOf` and `allOf`, but I don't see any connectors today that use those (only `oneOf` seems to be in use). So I don't think this should be a blocker of this PR but just wanted to note that it could be something we will want to add in the future.